tools/wpt/browser.py

```diff
@@ -431,6 +431,28 @@ def version(self):
         raise NotImplementedError
 
 
+class Safari(Browser):
+    """Safari-specific interface.
+
+    Includes installation, webdriver installation, and wptrunner setup methods.
```

Comment:
This is copypasta and just wrong, but so are most of the others. Maybe update it in a separate PR before this one.